trieste/pareto.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2020 The Trieste Contributors
```

Comment:
btw since you're not using Box, you could leave the file in utils. I don't really mind

Reply:
okay

Reply:
done